pillar.example

```diff
@@ -34,6 +34,12 @@ postgresql:
     wal_keep_segments: (unset)
     wal_buffers: (unset)
 
+    log_statement: 'none'
+    logging_collector: off
+    # NOTE: Only relevant when logging_collector is on
```

Comment:
Maybe add an additional newline before the `NOTE`, so it becomes clear the note is for both settings